src/legacy/core_plugins/vis_type_timeseries/public/components/vis_types/_vis_types.scss

```diff
@@ -7,4 +7,21 @@
   .tvbVisTimeSeries {
     overflow: hidden;
   }
+  .tvbVisTimeSeriesDark {
+    .echReactiveChart_unavailable {
+      color: #DFE5EF;
+    }
+    .echLegendItem  {
+      color: #DFE5EF;
+    }
+  }
+  .tvbVisTimeSeriesLight {
+    .echReactiveChart_unavailable {
+      color: #343741;
+    }
+    .echLegendItem  {
+      color: #343741;
+    }
+  }
```

Comment:
I could see this becoming an ever-growing list of overrides and we highly suggest not applying straight hex values when they come from the EUI palette/theme. We need to figure out a more extensible approach like possibly adding these to the Chart theme.

Reply:
This will be moved to elastic-charts for sure